activerecord/lib/rails/generators/active_record/migration.rb

```diff
@@ -24,7 +24,9 @@ def primary_key_type
         end
 
         def db_migrate_path
-          if defined?(Rails.application) && Rails.application
+          if migrations_paths = options[:migrations_paths]
```

Comment:
Is there any reason that named plural paths here even this `db_migrate_path` method returns a single path unlike other `migrations_paths`?

https://github.com/rails/rails/blob/ec387c6dd975fecea69db2ed9ff0c090ac59cf83/activerecord/lib/active_record/migration.rb#L1183
https://github.com/rails/rails/blob/cc81cd359c82bfba27388237a455fc345dce3c9f/activerecord/lib/active_record/tasks/database_tasks.rb#L84-L86

Reply:
Was just using the same name that's used everywhere else.